code/modules/food_and_drinks/drinks/drinks/bottle.dm

```diff
@@ -151,6 +153,48 @@
 	. = ..()
 	AddComponent(/datum/component/butchering, 200, 55)
 
+/obj/item/reagent_containers/food/drinks/bottle/imitation
+	name = "imitation glass bottle"
+	desc = "This very real glass bottle has the technology to morph its appearance based on its contents, but is ambigious on drinks outside its nano-software parameters."
+
+/obj/item/reagent_containers/food/drinks/bottle/imitation/on_reagent_change(datum/reagents/holder, ...)
+	. = ..()
+	if(!length(reagents.reagent_list))
+		renamedByPlayer = FALSE //so new drinks can rename the imitation bottle
+
+/obj/item/reagent_containers/food/drinks/bottle/imitation/update_name(updates)
+	if(renamedByPlayer)
+		return
+	. = ..()
+	var/datum/reagent/largest_reagent = reagents.get_master_reagent()
+	name = largest_reagent?.bottle_name || initial(name)
+
+/obj/item/reagent_containers/food/drinks/bottle/imitation/update_desc(updates)
+	if(renamedByPlayer)
+		return
+	. = ..()
+	var/datum/reagent/largest_reagent = reagents.get_master_reagent()
+	desc = largest_reagent?.bottle_desc || initial(desc)
+
+/obj/item/reagent_containers/food/drinks/bottle/imitation/update_icon_state()
+	if(!length(reagents.reagent_list))
+		icon_state = "glassbottle"
```

Comment:
not really important but
```suggestion
		icon_state = initial(icon_state)
```

Reply:
It already works, its a regular bottle when its not filled with recognised alcoholic reagents, but ill try this in a quick personal test.

Reply:
it is more in case someone decides to change the icon state